spec/requests/auth_no_email_spec.rb

```diff
@@ -60,6 +60,6 @@
     get "/auth/google_oauth2/callback"
     expect(response.status).to eq(302)
     data = JSON.parse(cookies[:authentication_data])
-    expect(data["username"]).to eq(nil)
+    expect(data["username"]).to eq("user1")
```

Comment:
Should the test name be updated too?

Reply:
It's still accurate - `user1` is not being suggested based on the email address. user# is core's new placeholder name.